sdk/helper/storagepacker/ids_test.go

```diff
@@ -0,0 +1,54 @@
+package storagepacker
+
+import (
+	"context"
+	log "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/sdk/logical"
+	"testing"
+)
+
+func getStoragePackerBits(tb testing.TB, baseBucketBits int, bucketShardBits int) *StoragePackerV2 {
+	storage := &logical.InmemStorage{}
+	storageView := logical.NewStorageView(storage, "packer/buckets/v2")
```

Comment:
I think this should be `logical.NewStorageView(storage, StoragePackerBucketsPrefix)`. `NewStoragePackerV2` operates on `config.BucketStorageView.Subview("v2/")` ([link](https://github.com/hashicorp/vault/pull/6724/files#diff-d460b49b4e599a77c84c96a74553b2d8R674)) so it's currently writing to `packer/buckets/v2/v2/`.

Reply:
You're correct, though it doesn't matter here (the redundant v2 isn't used, and would work fine even if we did write data.)

Reply:
fixed